driver/manifest/manifest.go

```diff
@@ -211,13 +180,58 @@ func Load(path string) (*Manifest, error) {
 	if err != nil {
 		return m, err
 	}
+	if err := LoadBuildManifest(m, open); err != nil {
+		return m, err
+	}
 	m.Maintainers, err = Maintainers(open)
 	if err != nil {
 		return m, err
 	}
 	return m, nil
 }
 
+func extractVersion(s string) string {
+	if i := strings.LastIndexByte(s, ':'); i > 0 {
+		s = s[i+1:]
+	}
+	if i := strings.LastIndexByte(s, '-'); i > 0 {
+		s = s[:i]
+	}
+	return s
+}
+
+func LoadBuildManifest(m *Manifest, open OpenFunc) error {
```

Comment:
Please document the exported function. In particular, that it modifies its argument (which is a little odd for an exported API).